component/widget/build.gradle.kts

```diff
@@ -1,11 +1,12 @@
 plugins {
 	id("project-module-android-library")
 	id("project-feature-paparazzi")
+	id("com.google.devtools.ksp")
```

Comment:
migrated to KSP to work around https://youtrack.jetbrains.com/issue/KT-67838 (Configuration.fileCollection(Spec) method has been deprecated)